specification/graphrbac/data-plane/Microsoft.GraphRbac/stable/1.6/graphrbac.json

```diff
@@ -1284,6 +1284,40 @@
         "x-ms-odata": "#/definitions/ServicePrincipal"
       }
     },
+    "/{tenantID}/servicePrincipalsByAppId/{applicationID}/objectId": {
+      "get": {
+        "tags": [
+          "ServicePrincipalsByAppId"
+        ],
+        "operationId": "ServicePrincipalsByAppId_Get",
+        "description": "Gets an object id for a given application id from the current tenant.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/tenantIDInPath"
+          },
+          {
+            "$ref": "#/parameters/applicationIdInPath"
```

Comment:
Please inline it (like the rest of the file) and call it simply `applicationId` (no InPath)

Reply:
Following tenantId naming convention. Won't fix for now.

Reply:
I insist, please inline this here and not ref an external definition to follow the consistency in this file. Thank!